src/guides/v2.4/graphql/authorization-tokens.md

```diff
@@ -4,7 +4,7 @@ title: Authorization tokens
 contributor_name: Atwix
 contributor_link: https://www.atwix.com/
 redirect_from:
-  - /guides/v2.3/graphql/get-customer-authorization-token.html
+  - /guides/v2.4/graphql/get-customer-authorization-token.html
```

Comment:
Remove the redirect. A 2.4 version of `get-customer-authorization-token.md` has never existed.

Reply:
It was symlinked - so the URL has existed. This is borne out by the fact if you visit  https://devdocs.magento.com/guides/v2.4/graphql/get-customer-authorization-token.html you get an empty page not a 404

Plus the new file - authorization-tokens.md has different 2.3 & 2.4 files. This file has also been complained about on issue #7844 
For safety I think this should stand.

Reply:
@keharper  Can you check this comment?  I think we can get this approved if we address this outstanding comment.